res/sql/create_v4.sql

```diff
@@ -19,4 +19,12 @@ CREATE TABLE auth_service_account (
     scopes TEXT,
     created_epoch INTEGER NOT NULL,
     modified_epoch INTEGER NOT NULL
-);
\ No newline at end of file
+);
+
+
+-- -----------------------------------------------------------------------
+-- SENDER
+-- TODO Sqlite doesn't support add constraint. Should us DROP/recreate table?
```

Comment:
What constraint do you need? If you do need a constraint then you can create a new table called sender_new with the column constraint. Then you can select from old table into new table to copy over the data, then you drop the old table and rename the new table

Reply:
That's the FK for the accounts table, linking each sender to a specific account. That's the way to do it with sqlite, drop/recreate...

Reply:
I will remove the account_id from the sender for now as we are going to keep the single account. It would take us to a bunch of decisions that we don't need to take now.